### DISCUSSION TITLE
Measuremet issue

### DISCUSSION THREAD

**Discussion by yitayalyonas** [winder/Universal-G-Code-Sender#2226] [General]
I am using ugs platform and my machine has 3 axis 
When joging y axis the left side y axis not repeat the first starting point so what will be the reason

**Comment by breiler** [comment]
I have no idea what you mean.

**Reply by yitayalyonas** [reply]
Ok thank you for your reply

**Comment by yitayalyonas** [comment]
Hi guys please guide me about main temrs of cnc machine specially on movement accuracy

**Reply by breiler** [reply]
Again, I have no idea what you mean - what do you mean by `temrs`?

Exactly what are you trying to achieve? 
What type of machine do you have and which firmware + version are you using?

Without details we will not be able to guide or help you.

**Comment by yitayalyonas** [comment]
My machine is cnc machine which have 3 axis and i am using ugs plateform

**Reply by breiler** [reply]
Repeating what you wrote in the first post does not make it more detailed...

By wasting a couple of minutes of my time I can try and guess from your vague description. You are jogging the Y axis a couple of millimeters in one direction and then try to jog back with the same distance. But the machine doesn't end up in the same physical place. Then this sounds like you are either missing steps or your machine is somehow slipping. 

So this problem is either a configuration error where you are trying to jog at a speed above the capabilities of your machine. Or you have a loose screw somewhere for example on your stepper motor couplers.

I am asking what firmware are your controller is running so that I can help you diagnose if it is a settings problem. But getting the settings from the controller depends on what firmware it is...

So which is it?
GRBL 1.1, GRBL 0.9, GRBL 0.8, TinyG, g2core, Smoothieware, GrblHAL, GRBL_ESP32, FluidNC

**Comment by yitayalyonas** [comment]
First thank you alot for replying
Excuse for my expression but you have got my idea

I am using GRBL1.1

**Reply by breiler** [reply]
Type the command `$$` to get the settings in the console and write the output here.

**Comment by yitayalyonas** [comment]
Thank you so much !
The problem has solved as you said
And you are generous and helpfull person

Good luck!
